### PR TITLE
Feat: Support Running Sidecar with a Command.

### PR DESCRIPTION
**Pull request by tabbyl21**
This PR addresses issue #2448 . Some containers may not have entry points, if this is the case they would need to be run using a command. This change extends the definition of sidecar so that there is an optional command field. If the field is present then the container will be run using that command. This is a two line change that is fully backward compatible.

### CODE REVIEW

**Comment by tabbyl21** [comment]
Hi @Jan-M @jopadi ,

I am wondering what I need to do to get this PR accepted into your project! I believe I have followed the guidelines from the ReadME to the extent its applicable! Please let me know if there is anything else I need to add!

Thank you for your time,
Tabatha

**Comment by AleksandrPaul** [comment]
Hi @Jan-M , @sdudoladov , @FxKu , @idanovinda , @hughcapet, @jopadi 

Could you please review this PR?

**Comment by sgielen** [comment]
Hi @FxKu could you please review this PR? It is similar to a PR you recently reviewed. Thanks

**Comment by kamenskiyyyy** [comment]
Any updates?

File: pkg/apis/acid.zalan.do/v1/postgresql_type.go [hunk 1]
```diff
@@ -220,6 +220,7 @@ type Sidecar struct {
 	DockerImage string             `json:"image,omitempty"`
 	Ports       []v1.ContainerPort `json:"ports,omitempty"`
 	Env         []v1.EnvVar        `json:"env,omitempty"`
+	Command     []string           `json:"command,omitempty" protobuf:"bytes,3,rep,name=command"`
```
Comment (FxKu) [inline pkg/apis/acid.zalan.do/v1/postgresql_type.go:223]: why is `protobuf:"bytes,3,rep,name=command"` necessary? We have it nowhere else. Is it similar to filter inputs using pattern in the CRD schema?

**Comment by FxKu** [comment]
@tabbyl21 sorry for not reacting for a looong time. We are usually a bit reluctant to changes that offer more config mean and adding complexity. After taking a closer look I get now why this change is needed. Being able to run any command via sidecar just sounded too scary to me first (well, maybe because it is). If you could answer my one question we could merge it then.

Try to run codegen (./hack/update-codegen.sh) to see if something changes.

**Comment by icc-garciaju** [comment]
One question here.
Why only "command" and not include "args" too?

**Comment by sgielen** [comment]
> One question here. Why only "command" and not include "args" too?

It is a []string so it includes the args, too

File: pkg/apis/acid.zalan.do/v1/postgresql_type.go (hunk 1 above)
Comment (tabbyl21) [reply pkg/apis/acid.zalan.do/v1/postgresql_type.go:223]: apologies just seeing this comment. I can remove that part

**Comment by tabbyl21** [comment]
@FxKu apologies didn't see your repsonses! I have addressed your comment! Let me know what changes need to be made to merge this in!

**Comment by hughcapet** [comment]
👍

**Comment by FxKu** [comment]
:+1:

**Pull request merged**